arangod/Agency/Job.cpp

```diff
@@ -428,10 +457,10 @@ void Job::doForAllShards(Node const& snapshot, std::string& database,
     std::string curPath =
```

Comment:
This shadows the argument, so the actual argument is never taken. This cannot be intended, or?

Reply:
Not it's not :)